assets/sass/v2/_password.scss

```diff
@@ -0,0 +1,12 @@
+.password-authenticator {
+  &__rules-heading {
+    margin-bottom: 1rem;
```

Comment:
Maybe not for this PR, but are we using Odyssey standards for v2? If so, can we use standard spacing units from Odyssey? If not, might be worth it to consolidate the spacing units in one file and then re-use. Eventually, we can convert from that one place to design systems when we move.

Reply:
This has nothing to do with Odyssey, it's just BEM naming convention - http://getbem.com/introduction/ , I believe Odyssey has borrowed the standard. It gives a clean way of writing css class names - helps in improving css code when used with sass/scss.

Reply:
Also, it's being used in different places in SIW (I probably would be a major contributor to that so far :) )

Reply:
@santhoshbalakrishnan correct me wrong, the naming convention in Odyssey is using `-` and `--` but not `__`

Reply:
^ Exactly what @haishengwu-okta said. We already have a naming convention(even though not documented) of using `-` and `--` in class names. We can consolidate and use a single version.

But, I'm glad that you care and have chosen to follow some good naming conventions for CSS :)

Reply:
Updated